lib/github-pages-health-check/domain.rb

```diff
@@ -286,6 +293,11 @@ def a_record?
         dns.first.type == Dnsruby::Types::A
       end
 
+      def aaaa_record?
+        return unless dns?
+        dns.index { |r| r.type == Dnsruby::Types::AAAA }
```

Comment:
Would `dns.any?` make better sense here? It returns a boolean value which is what I think you want here.